### DISCUSSION TITLE
[Feature Request]: 更改默认下载器保存后不生效。必须重启后生效

### DISCUSSION THREAD

**Discussion by cng10001** [jxxghp/MoviePilot#3484] [讨论]
### 当前程序版本

1.8.8

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

![image](https://github.com/jxxghp/MoviePilot/assets/115156545/7dd44309-e9e3-4547-b172-a1902400d98f)
这个设置。
**能否优化下，或者提示需重启后生效**

**Comment by cng10001** [comment]
提示模块重新载入成功  实则需要重启后才会生效

**Comment by Taurus0514** [comment]
一样的问题，在MP界面设置里面更改下载器（QB更换TR），然后下载的时候提示未知错误不能下载，只能在docker compose yaml中修改下载器设置才可以使用
